pages/login.tsx

```diff
@@ -7,6 +7,7 @@ import { DarkModeToggle } from "../Components/components";
 import { AuthContext } from "@/context/AuthContext";
 import { ToastContainer, toast } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
+import "./signup";
```

Comment:
Why do we need this?

Reply:
@lalitmee by mistake, I had added this file but I removed it now